db/fixtures/miq_product_features.yml

```diff
@@ -5095,6 +5095,11 @@
       :description: View Ansible Tower Job Templates
       :feature_type: view
       :identifier: configuration_script_view
+      :children:
+      - :name: List
+        :description: List Configuration Script Sources
+        :feature_type: view
+        :identifier: configuration_script_sources_list
```

Comment:
@jntullo, i think it's better to wait, these features will need to be moved under new location once https://github.com/ManageIQ/manageiq/pull/13526 is merged